rust/agents/validator/src/validator.rs

```diff
@@ -222,6 +236,7 @@ impl Validator {
                         validator_address=?announcement.validator,
                         "Please send tokens to the validator address to announce",
                     );
+                    sleep(self.interval).await;
```

Comment:
Why move this?

Reply:
otherwise the validator sleeps after successful announcement which seems silly